tasks/41_sort-by-bits/sort-by-bits.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+
+export const sortByBits = (arr) => {
+  const copy = arr.slice();
+  return copy.sort( (a, b) => {
+    let bitDifference = a.toString(2).replace(/0/g, '').length - b.toString(2).replace(/0/g, '').length;
```

Comment:
instead of disabling max lenth you can do
let bitDifference = a.toString(2)
   .replace(/0/g, '')
   .length

Or you can split this into few variables to make code more readable